docs/source/contributing.rst

```diff
@@ -318,7 +318,7 @@ format all our code to a standard style. While you're editing code you
 can be as sloppy as you like about whitespace; and then before you commit,
 just run:
 
-.. code-block::
+.. code-block:: text
 
     pip install -U pre-commit
     pre-commit
```

Comment:
I'd personally do `.. code-block:: sh` or similar when code is intended for use at a shell prompt, but we don't really need syntax highlighting or anything so I don't think it matters